### ISSUE TITLE
config file, value missing

### ISSUE THREAD

**Issue by Shiro-LK** [feymanpriv/DOLG-paddle#4]
Hello,

Thank you for sharing the architecture of DOLG.

I have few questions regarding the implementation and the config file.
I saw in the architecture that there is no final layer used during training. Does the final descriptor is just followed by a linear layer with a softmax and arcface module ? 
A question about the config Yaml, `STRIDE_1X1 = False`, but in the core.config it is written : ` # Apply stride to 1x1 conv (True -> MSRA; False -> fb.torch)`
`_C.RESNET.STRIDE_1X1 = True`
Does that mean, I need to change in the yaml STRIDE_1X1 to True as we are using paddle and not pytorch ?
Thanks in advance for your help!

**Comment by feymanpriv** [comment]
"I saw in the architecture that there is no final layer used during training. Does the final descriptor is just followed by a linear layer with a softmax and arcface module ?" 
Yes, this is inference code here, you can email me for the training codes written in torch.

A question about the config Yaml, STRIDE_1X1 = False, but in the core.config it is written : # Apply stride to 1x1 conv (True -> MSRA; False -> fb.torch)
_C.RESNET.STRIDE_1X1 = True
Does that mean, I need to change in the yaml STRIDE_1X1 to True as we are using paddle and not pytorch ?
You can see the model/resnet.py
